### PR TITLE
feat(KUI-1265): upgrade to KTH Style 10

### PR DESCRIPTION
**Pull request by karlandindrakryggen**
This PR upgrades KTH Style to version 10 and includes some more cleanup and fixes:

- Upgrade to KTH style packages 
  - Includes major cleanup of CSS
  - Adding same shared concept as in public web apps with copied files in `_shared.scss` and `components-shared` folder 

https://kth-se.atlassian.net/browse/KUI-1265

### CODE REVIEW

**Comment by karlandindrakryggen** [comment]
This PRs is more or less the same as the other admin web upgrades, but the cleanup has not been that thoroughly and some other fixes (as favicon and toolbar env var) were skipped here since `kursutveckling-admin-web` probably want live for long time.
- kurs-pm-admin-web: https://github.com/KTH/kurs-pm-admin-web/pull/276
- kurs-pm-data-admin-web: https://github.com/KTH/kurs-pm-data-admin-web/pull/368
- kursinfo-admin-web https://github.com/KTH/kursinfo-admin-web/pull/273

File: i18n/messages.se.js [hunk 1]
```diff
@@ -116,8 +117,8 @@ module.exports = {
     /** ************ PAGE 2 FORM ************** */
     asterix_text: 'Kommer inte att kunna ändras efter publicering',
```
Comment (belanglos) [inline i18n/messages.se.js:118]:
```suggestion
    asterisk_text: 'Kommer inte att kunna ändras efter publicering',
```

Comment (belanglos) [reply i18n/messages.se.js:118]: 🥲

**Review by belanglos — APPROVED** [review]
LGTM!
If you want to s/asterix/obelix go ahead! ;-)

File: i18n/messages.se.js (hunk 1 above)
Comment (karlandindrakryggen) [reply i18n/messages.se.js:118]: Menar du? :D 
```suggestion
    obelix_text: 'Kommer inte att kunna ändras efter publicering',
```

**Pull request merged**